widget/embedded/src/components/common/ActivateTabAlert/ActivateTabAlert.tsx

```diff
@@ -0,0 +1,24 @@
+import type { PropTypes } from './ActivateTabAlert.types';
+
+import { i18n } from '@lingui/core';
+import { Alert, Button } from '@rango-dev/ui';
+import React from 'react';
+
+export function ActivateTabAlert(props: PropTypes) {
+  return (
+    <Alert
+      action={
+        <Button
+          onClick={props.onActivateTab}
+          variant="contained"
+          size="xxsmall"
+          type="warning">
+          {i18n.t('activate this tab')}
```

Comment:
`Activate this tab.`

Reply:
I also suggest this text be `Activate` and description be `Another tab is open and handles transactions. You can activate this tab.`